### ISSUE TITLE
Populated values are removes after `save()`.

### ISSUE THREAD

**Issue by dangreen** [balderdashy/sails#3759]
**Sails version**: v0.12.3
**Node version**: v5.11.0
**NPM version**: 3.8.6
**Operating system**: OS X

<hr/>
```js
var invoice = await Invoice.findOne(id).populate("subscriptions");

var { subscriptions } = invoice; // length is 1

invoice.paid = true;
await invoice.save();
// after `save()` subscriptions is empty array
```

**Comment by tbjers** [comment]
I think this is intended behavior, albeit somewhat overzealous. The model now wipes all attributes that are not part of the model upon save. However, relational data should probably be preserved.

**Comment by sailsbot** [comment]
@dangreen,@tbjers: Hello, I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message and simply close the issue if possible.  On the other hand, if you are still waiting on a patch, please post a comment to keep the thread alive (with any new information you can provide).



If no further activity occurs on this thread within the next 3 days, the issue will automatically be closed.

Thanks so much for your help!

**Comment by CXRom** [comment]
@tbjers I'm agree with you, If they can add an option param for active the population it'll be grate.

**Comment by sailsbot** [comment]
@dangreen,@tbjers,@sailsbot,@CXRom: Hello, I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message and simply close the issue if possible.  On the other hand, if you are still waiting on a patch, please post a comment to keep the thread alive (with any new information you can provide).



If no further activity occurs on this thread within the next 3 days, the issue will automatically be closed.

Thanks so much for your help!
